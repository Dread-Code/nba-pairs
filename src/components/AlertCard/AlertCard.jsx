import React from 'react'
import { Card } from 'semantic-ui-react'
import PropTypes from 'prop-types'
import './AlertCard.scss'

const AlertCard = ({ error }) => (
  <Card>
    <Card.Content className="alert-card" header="Alert" />
    <Card.Content className="alert-card" description={error} />
  </Card>
)
AlertCard.propTypes = {
  error: PropTypes.string.isRequired
}

export default AlertCard
