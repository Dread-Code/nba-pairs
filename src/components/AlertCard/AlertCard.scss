.alert-card {
  background: #f47575 !important;
}

.alert-card:last-of-type {
  background: #db2828e6 !important;
}
