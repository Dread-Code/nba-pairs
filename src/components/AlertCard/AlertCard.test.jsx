import React from 'react'
import { shallow } from 'enzyme'
import AlertCard from './AlertCard'

describe('AlertCard', () => {
  const errorText = 'No matches found'
  const wrapper = shallow(<AlertCard error={errorText} />)
  test('1) should match snapshoot ', () => {
    expect(wrapper).toMatchSnapshot()
  })
  test('2) should return the text in CardContent description', () => {
    expect(expect(wrapper.find('CardContent').getElements()[1].props.description).toBe(errorText))
  })
})
