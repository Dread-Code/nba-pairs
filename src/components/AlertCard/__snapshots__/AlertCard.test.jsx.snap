// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertCard 1) should match snapshoot  1`] = `
<Card>
  <CardContent
    className="alert-card"
    header="Alert"
  />
  <CardContent
    className="alert-card"
    description="No matches found"
  />
</Card>
`;
